packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -97,17 +97,16 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
   }
 
   async getConfigurationStream(): Promise<Writable> {
-    if (!this.strapi) {
-      throw new Error('Not able to stream Configurations. Strapi instance not found');
-    }
-
-    const strapi = this.strapi;
-
     return new Writable({
       objectMode: true,
-      async write(config, _encoding, callback) {
+      write: async (config: IConfiguration<any>, _encoding, callback) => {
+        if (!this.strapi) {
```

Comment:
This check will only be run after the transfer has started, is there a reason why you moved that here?

Reply:
Aesthetics, also because we'll perform a transfer regardless if it's a merge or a restore, it's just that we only have restore for now.

Reply:
oh wait i guess you are speaking about the this.strapi check

Reply:
well Typescript will throw an error if I do the check outside, but yeah it does make more sense before starting the streaming

Reply:
Yes, I was talking about the this.strapi check. We need to do it before the transfer actually start, but nothing prevents you from doing it twice: once before streaming & once when starting to stream data.